c2corg_ui/static/js/map/search.js

```diff
@@ -132,6 +132,7 @@ app.MapSearchController.SEARCH_URL = 'https://photon.komoot.de/api/';
 app.MapSearchController.prototype.createAndInitBloodhound_ = function() {
   let url = app.MapSearchController.SEARCH_URL;
   url += '?q=%QUERY';
+  url += '&osm_tag=place:city&osm_tag=place:town&osm_tag=place:village';
```

Comment:
This should rather be written in a single line, with query in the end (although it's only cosmetics)

`url += '?osm_tag=place:city&osm_tag=place:town&osm_tag=place:village&q=%QUERY';`